Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -567,7 +566,7 @@ public int FirstHiddenCellIndex
                 }
             }
         }
-
+        
```

Comment:
Other whitespace. Maybe worth formatting this file in VS?

Reply:
Fixed!